src/test_l1_dsHdmiIn.c

```diff
@@ -1649,21 +1694,22 @@ void test_l1_dsHdmiIn_negative_dsIsHdmiARCPort_sink(void) {
  * 
  */
 void test_l1_dsHdmiIn_positive_dsGetEDIDBytesInfo_sink(void) {
+
     gTestID = 33;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     uint8_t numInputPorts = 0;
     unsigned char* edidBytes1 = (unsigned char*) malloc(255);
     unsigned char* edidBytes2 = (unsigned char*) malloc(255);
     unsigned char* edidBytes3 = (unsigned char*) malloc(255);
     unsigned char* edidBytes4 = (unsigned char*) malloc(255);
-    int edidSize1;
-    int edidSize2;
-    int edidSize3;
-    int edidSize4;
+    int edidSize1 = 0;
+    int edidSize2 = 0;
+    int edidSize3 = 0;
+    int edidSize4 = 0;
 
     // Step 1: Initialize the HDMI input sub-system using dsHdmiInInit()
     UT_ASSERT_EQUAL_FATAL(dsHdmiInInit(), dsERR_NONE);
-   
+
     // Step 2: Get the number of input ports from the profile file
     numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/HdmiInputPort/numberOfPorts");
     if (numInputPorts > 0) {
```

Comment:
if (edidBytes1 == NULL || edidBytes2 == NULL || edidBytes3 == NULL || edidBytes4 == NULL) {
        UT_LOG("\n Memory allocation failed in %s\n", __FUNCTION__);
    } else {
    
    add the numInputPorts code inside the else case
    }